File: packages/twilio_flutter_video/ios/Classes/Bridge/VideoEventEmitter.swift
import Foundation

final class VideoEventEmitter {
  private let flutterApi: TwilioVideoFlutterApi

  init(binaryMessenger: FlutterBinaryMessenger) {
    flutterApi = TwilioVideoFlutterApi(binaryMessenger: binaryMessenger)
  }

  func emit(event: VideoNativeEvent) {
    flutterApi.onNativeEvent(event: event) { _ in }
  }
}

File: packages/twilio_flutter_video/android/src/main/kotlin/com/twilioflutter/video/bridge/VideoEventEmitter.kt
package com.twilioflutter.video.bridge

import com.twilioflutter.video.pigeon.TwilioVideoFlutterApi
import com.twilioflutter.video.pigeon.VideoNativeEvent
import io.flutter.plugin.common.BinaryMessenger

class VideoEventEmitter(binaryMessenger: BinaryMessenger) {
  private val flutterApi = TwilioVideoFlutterApi(binaryMessenger)

  fun emit(event: VideoNativeEvent) {
    flutterApi.onNativeEvent(event) {}
  }
}
